charts/tidb-operator/templates/NOTES.txt

```diff
@@ -1,7 +1,5 @@
-1. Make sure tidb-operator components are running
-   kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/instance={{ .Release.Name }}
-2. Install CRD
-   kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml
-   kubectl get customresourcedefinitions
-3. Modify tidb-cluster/values.yaml and create a TiDB cluster by installing tidb-cluster charts
-   helm install tidb-cluster
+First, make sure tidb-operator components are running:
+
+    kubectl get pods --namespace {{ .Release.Namespace }} -l app.kubernetes.io/instance={{ .Release.Name }}
+
+Then follow the next steps to create your TiDB clusters.
```

Comment:
What steps?

Reply:
These are the documents the user is reading. For example:

https://pingcap.com/docs/tidb-in-kubernetes/stable/get-started/#deploy-tidb-cluster
https://pingcap.com/docs/tidb-in-kubernetes/stable/deploy-on-general-kubernetes/

Reply:
I think we can remove this, users can go to the next steps following the doc.